submissions/adeyemiJuwon/todo/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Form from './Form';
+import Todos from './Todos';
+import './App.css';
+
+export default class App extends Component {
+  state = {
+    todos: [],
+  };
+  addtodo = (todo) => {
+    const newTodo = {
+      id: new Date().getTime().toString(),
```

Comment:
Why are you doing this?

Reply:
Instead of install id generator package, I improvise  by using the time of every input as id

Reply:
There is no need to use an id generator. You can use the position of the todo in the array.